svg/renderer.go

```diff
@@ -397,3 +365,57 @@ func (r *SVG) RenderImage(img image.Image, m canvas.Matrix) {
 	r.writeClasses(r.w)
 	fmt.Fprintf(r.w, `"/>`)
 }
+
+func writeImageBytes(img image.Image, mimetype string, w io.Writer) error {
+	if j, ok := img.(canvas.JPEGImage); ok {
+		_, err := w.Write(j.JPEGBytes())
+		return err
+	}
+	if p, ok := img.(canvas.PNGImage); ok {
+		_, err := w.Write(p.PNGBytes())
+		return err
+	}
+	if mimetype == "image/jpg" {
+		return jpeg.Encode(w, img, nil)
+	}
+	return png.Encode(w, img)
```

Comment:
We need to check for mimetype explicitly I think, we might add other file types in the future.